spec/builtins.js

```diff
@@ -47,6 +47,22 @@ describe('builtin helpers', function() {
       var string = '{{#with person}}Person is present{{else}}Person is not present{{/with}}';
       shouldCompileTo(string, {}, 'Person is not present');
     });
```

Comment:
@kpdecker, I hope the specs can also serve as a more detailed summary of the changes.